src/d3d11/d3d11_device.h

```diff
@@ -428,6 +429,22 @@ namespace dxvk {
 
     bool Is11on12Device() const;
 
+    void AddSwapchain(D3D11SwapChain* swapchain) {
+      m_swapchains.push_back(swapchain);
+    }
+
+    void RemoveSwapchain(D3D11SwapChain* swapchain) {
+      std::remove(m_swapchains.begin(), m_swapchains.end(), swapchain);
```

Comment:
```suggestion
      m_swapchains.erase(std::remove(m_swapchains.begin(), m_swapchains.end(), swapchain));
```

Found by clang-cl.

Reply:
Fixed, thanks for running this change through clang-cl!